pkgs/by-name/ca/capnproto/package.nix

```diff
@@ -40,12 +44,25 @@ clangStdenv.mkDerivation rec {
     hash = "sha256-aDcn4bLZGq8915/NPPQsN5Jv8FRWd8cAspkG3078psc=";
   };
 
-  nativeBuildInputs = [ cmake ];
+  patches = [
+    # https://github.com/capnproto/capnproto/pull/2377
+    (fetchpatch {
+      url = "https://github.com/capnproto/capnproto/pull/2377/commits/f26dd335c8650a2f8ab7d6e4fb5dfc40ee6af618.patch";
```

Comment:
This is prone to garbage collection. Merge needs to be awaited or the patch vendored.